orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/PipelineStarterListener.groovy

```diff
@@ -76,11 +81,13 @@ class PipelineStarterListener implements JobExecutionListener {
             def queuedExecution = executionRepository.retrievePipeline(id)
             log.info("starting pipeline ${nextPipelineId} due to ${execution.id} ending")
             pipelineStarter.startExecution(queuedExecution)
-          } else {
+            startTracker.removeFromQueue(execution.pipelineConfigId, id)
```

Comment:
not too keen on this being a global flag. Feels like you want to also be able to override this on a per pipeline config level.